core/mixins/_media-queries.scss

```diff
@@ -97,3 +105,16 @@
     @content;
   }
 }
+
+
+/**
+ * Retina.
+ */
+
+@mixin respond-retina($resolution: $retina-resolution) {
+  $resolution: if(unitless($resolution), $resolution * 96dpi, $resolution);
+  
+  @media (min-resolution: $resolution) {
+    @content;
+  }
+}
```

Comment:
> OK best not to mention them at all then? Or provide a warning?

I wouldn't even mention them. The `dpi` value will be supported for ages across the board. If and when the other units get better support in browsers and prefixers, then mention. I'd bet that most devs probably won't even know they exist…